nengo_viz/static/viz_modal.js

```diff
@@ -61,6 +62,19 @@ VIZ.Modal.prototype.text_body = function(text, type) {
     $p.append(document.createTextNode(text));
 }
 
+VIZ.Modal.prototype.help_body = function() {
+    this.clear_body();
+
+    this.$div.find('.modal-dialog').addClass('modal-sm');
+    var $body = $('<table width=100%>');
+    $body.append('<tr> <td>Play / pause</td> <td align="right">Spacebar</td></tr>');
+    $body.append('<tr> <td>Undo</td> <td align="right">Ctrl-z</td></tr>');
+    $body.append('<tr> <td>Redo</td> <td align="right">Ctrl-Shift-z || Ctrl-y</td></tr>');
```

Comment:
Only huge nerds will know what `||` means. How about the word or? If you want it to stand out, you could put or in italics or bold.